apikit/convenience.py

```diff
@@ -335,6 +423,25 @@ def __init__(self, name, version, repository, description,
                            api_version=api_version,
                            auth=auth,
                            route=route)
+        logfile = None
+        syslog = False
+        loghost = None
+        loglevel = None
+        if "LOGFILE" in os.environ and os.environ["LOGFILE"]:
```

Comment:
All of this environment variables <=> logging configuration could be in get_logger. Then using it could be generic for all of our python projects.